src/Raven.Server/Documents/Handlers/Processors/Revisions/RevisionsHandlerProcessorForRevertRevisions.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using JetBrains.Annotations;
+using Raven.Server.Documents.Operations;
+using Raven.Server.Documents.Revisions;
+using Raven.Server.ServerWide;
+using Raven.Server.ServerWide.Context;
+
+namespace Raven.Server.Documents.Handlers.Processors.Revisions
+{
+    internal class RevisionsHandlerProcessorForRevertRevisions : AbstractRevisionsHandlerProcessorForRevertRevisions<DatabaseRequestHandler, DocumentsOperationContext>
+    {
+        public RevisionsHandlerProcessorForRevertRevisions([NotNull] DatabaseRequestHandler requestHandler) : base(requestHandler)
+        {
+        }
+
+        protected override void RevertRevisions(long operationId, RevertRevisionsRequest configuration, OperationCancelToken token)
+        {
+            var t = RequestHandler.Database.Operations.AddLocalOperation(
```

Comment:
don't you need to do the continuation and dispose the token? Check here: https://github.com/ravendb/ravendb/blob/v6.0/src/Raven.Server/Documents/Sharding/Handlers/Processors/Queries/AbstractShardedOperationQueriesHandlerProcessor.cs#L59

Reply:
this seems to already happen inside? https://github.com/ravendb/ravendb/blob/ffe03341cb20df1a9ac8e1d4b187bd61909de88c/src/Raven.Server/Documents/Operations/AbstractOperations.cs#L71